src/puppetlabs/puppetdb/http/index.clj

```diff
@@ -0,0 +1,18 @@
+(ns puppetlabs.puppetdb.http.index
+  (:require [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.http.query :as http-q]
+            [puppetlabs.puppetdb.middleware :refer [wrap-with-paging-options]]
+            [puppetlabs.puppetdb.query.paging :as paging]))
+
+(defn routes
+  [version optional-handlers]
+  (let [param-spec {:optional paging/query-params
+                    :required ["query"]}]
+    (app
```

Comment:
Meh, I think new endpoints have been compojure. That's certainly where we want to go. I think this would be pretty simple to make compojure as well, just adding work for ourselves later if we don't.